src/Form.js

```diff
@@ -0,0 +1,199 @@
+import React, { useState, useRef } from "react";
+import "./Form.css";
+const Form = () => {
+  const [formData, setFormData] = useState({
+    loginId: "",
+    email: "",
+    name: "",
+    timezone: "",
+    homePage: "",
+    aboutMe: "",
+    receiveNotifications: false
+  });
```

Comment:
create a default form fields and move that outside the component
```suggestion
  const [formData, setFormData] = useState(defaultFormField);
```